package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "migrate-mongo",
-  "version": "10.0.0",
+  "version": "10.0.1",
```

Comment:
This is just bug fixing? I think we should update minor version. As we are removing package.